src/main/java/org/candlepin/hostedtest/HostedTestResource.java

```diff
@@ -263,9 +273,9 @@ public OwnerInfo createOwner(OwnerDTO owner) {
     @Consumes(MediaType.APPLICATION_JSON)
     @Produces(MediaType.APPLICATION_JSON)
     @Path("/subscriptions")
-    public SubscriptionInfo createSubscription(
+    public SubscriptionDTO createSubscription(
```

Comment:
PoolDTO wasn't a fit here?

Reply:
This might need a separate task. I think it will work but it needs a lot of changes (mostly in children creation and ruby tests) due to the differences in how PoolDTO and SubscriptionDTO handles producs.

Reply:
ack